src/components/narrative/MobileNarrativeDisplay.js

```diff
@@ -89,9 +89,7 @@ class MobileNarrativeDisplay extends React.Component {
     }
     const block = this.props.blocks[this.props.currentInFocusBlockIdx];
     return (
-      <div
-        id={`MobileNarrativeBlock_${this.props.currentInFocusBlockIdx}`}
```

Comment:
This is used for scrolling to a particular block via the URL query parameter. While this doesn't always work (bugs separate to this work) I'd like to preserve these id's here please.